requirements.txt

```diff
@@ -14,4 +14,6 @@ imagesize
 pyproj
 pygeotile
 typing-extensions
-pytest-xdist
\ No newline at end of file
+pytest-xdist
+nbformat~=5.7.0
```

Comment:
will this address the codefresh failues like https://g.codefresh.io/build/63ee46d0b77e2079fc9625a2?
But I assume nbformat is not required for development... if this is true, we should do smth like this https://realpython.com/lessons/production-vs-development-dependencies/

PS We need to switch to poetry asap

Reply:
Yes, this is meant to fix codefresh failures. nbformat is required to run examples/test_notebooks.py, so its required for development. Good idea to split production vs dev requirements, but isn't that already the case since package requirements are listed in `setup.py` and dev requirements in `requirements.txt`?